include/envoy/common/platform.h

```diff
@@ -53,13 +53,13 @@
   __pragma(pack(push, 1)) definition, ##__VA_ARGS__;                                               \
   __pragma(pack(pop))
 
-using ssize_t = ptrdiff_t;
+typedef ptrdiff_t ssize_t;
```

Comment:
nit: I believe we bias to using `using`.

Reply:
We are aware of this and wrote the original solve. We've run across an edge case where using was insufficient, and these are global C declarations of posix to be emulated correctly on Windows  (We did joke about making it a #define, obviously not seriously:)

Reply:
I believe we changed those to `typedef` instead of `using` to appease clang-tidy in this commit: https://github.com/envoyproxy/envoy/pull/11565/commits/6ab3de8b3afc8c849bbd80ebf9ae05b3d55cb357, if there is a clang-tidy trick we don't know of (besides just turning tidy off for that section) we could, change to, that would be great to know